### PR TITLE
Markdown corrections, Azure code examples

### PR DESCRIPTION
**Pull request by AErmie**
- Fixed markdown errors
- Added working code examples for Azure
- Unified Azure code syntax highlighting to use Bash (since the instructions are for Azure Cloud Shell)

### CODE REVIEW

File: .github/template.md [hunk 1]
```diff
@@ -165,4 +189,95 @@ We would love to hear about more ideas on how to find vulnerable infrastructure-
 
 If you need direct support you can contact us at [info@bridgecrew.io](mailto:info@bridgecrew.io).
 
-# Existing vulnerabilities (Auto-Generated)
+## Existing vulnerabilities (Auto-Generated)
```
Comment (nimrodkor) [inline .github/template.md:192]: @AErmie This part is auto-generated, so please don't override anything after this title - just leave it as was before :)

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
-# TerraGoat - Vulnerable Terraform Infrastructure 
+# TerraGoat - Vulnerable Terraform Infrastructure
```
Comment (nimrodkor) [inline README.md:1]: @AErmie This entire file will be overwritten by template + auto-generated part, so all the changes here are irrelevant

Comment (AErmie) [reply README.md:1]: Thanks, but I'm not sure how to remove the ReadMe file from the Pull Request after it' been submitted.

File: .github/template.md (hunk 1 above)
Comment (AErmie) [reply .github/template.md]: I'm not sure how to revert the one section. But, in the auto-generated part, you should change the heading accordingly as it causes a markdown syntax issue.

**Comment by nimrodkor** [comment]
np. Thanks @AErmie !

**Pull request merged**